spec/markdown_attachment_processor_spec.rb

```diff
@@ -83,24 +83,4 @@
       end
     end
   end
-
-  describe "#attributes" do
-    before do
-      allow(doc).to receive(:attributes).and_return(doc_attrs)
-    end
-
-    let(:doc_attrs) {
-      {
-        body: unprocessed_body,
-        other_stuff: "other stuff",
-      }
-    }
-
-    it "merges the rendered body into the attributes" do
-      expect(renderer.attributes).to eq(
-        body: processed_body,
-        other_stuff: "other stuff",
-      )
-    end
-  end
```

Comment:
Minor: You could consider squashing this into the commit where it stopped being used.